source/templates/committer-welcome.txt

```diff
@@ -0,0 +1,67 @@
+To: private@[PROJECT].a.o, NewCommitterID@apache.org
+Subject: Welcome, New Committer!
+
+[Committer Name] as you know, the ASF Infrastructure has set up your
+committer account with the username `[apacheID]`
+
+[If your project uses SVN]
+
+Please follow the instructions to set up your SSH,
+svn password, svn configuration, email forwarding, etc.
+https://www.apache.org/dev/#committers
+
+[If your project automatically adds committers as PMC members]
+Please subscribe to the [PROJECT] Project Management
+Committee mailing list private@[PROJECT].apache.org.
+[/If PMC]
+
+[PROJECT] has various resources at:
+  https://svn.apache.org/repos/asf/[PROJECT]
+
+[/If SVN]
+
+[If your project uses GitHub]
+You will now be able to merge approved PRs on GitHub for this project.
+
+You can manage your account settings at https://id.apache.org/
+[/If GitHub]
+
+As an ASF committer, you now also have commit access to specific
+sections of the ASF Foundation repository, as follows:
+
+The general "committers" at:
+https://svn.apache.org/repos/private/committers
+
+The developer section of the website describes roles within the ASF and 
+provides other resources:
+  https://www.apache.org/foundation/how-it-works.html
+  https://www.apache.org/dev/
+
+The incubator also has some useful information for new committers
+in incubating projects:
+  https://incubator.apache.org/guides/committer.html
+  https://incubator.apache.org/guides/ppmc.html
+
+Just as before you became a committer, participation in any ASF community
+requires adherence to the ASF Code of Conduct:
+  https://www.apache.org/foundation/policies/conduct.html
+
+[PROJECT should insert its own guidelines here; if none are available,
+ the Apache Forrest guidelines are available as a template.]
+  https://forrest.apache.org/guidelines.html
```

Comment:
Mentioning forrest here is not really good I think - because the project is in the attic:

<img width="2056" alt="Screenshot 2024-10-27 at 22 08 30" src="https://github.com/user-attachments/assets/6e56bef0-7c65-44e8-bf4a-7eb031cad866">

Also:

<img width="253" alt="Screenshot 2024-10-27 at 22 10 01" src="https://github.com/user-attachments/assets/c12cbdf0-f115-430d-a809-f9c11d13c32d">

Not sure if we want to give the impression to projects that they shoudl base their guidelines on a project that has been retired many years ago.

I could propose Airflow's https://github.com/apache/airflow/blob/main/COMMITTERS.rst but it's a bit more than guidelines - but possibly there are other project guidelines that might be used here.

Reply:
Yeah, that, also, was content that has been in this page forever. It's on my list to find a better project to reference.